src/calculateRentalCost.js

```diff
@@ -1,10 +1,25 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
```

Comment:
```suggestion
  const PRICE_PER_DAY = 40;
  const LONG_TERM = 7;
  const LONG_TERM_DISCOUNT = 50;
  const SORT_TERM = 3;
  const SORT_TERM_DISCOUNT = 20;
```
Consider using naming conventions for naming constants